architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -11,11 +11,22 @@
 toc::[]
 
 == Builds
-A build is the process of transforming input parameters into a resulting object.
-Most often, the process is used to transform source code into a runnable image.
-A link:../../dev_guide/builds.html#defining-a-buildconfig[BuildConfig] object is
+A link:../../dev_guide/builds.html[build] is the process of transforming input
+parameters into a resulting object. Most often, the process is used to transform
+input parameters or source code into a runnable image. A
+link:../../dev_guide/builds.html#defining-a-buildconfig[BuildConfig] object is
 the definition of the entire build process.
 
+OpenShift leverages Kubernetes by creating Docker containers from build images
+and pushing them to a
+link:../../install_config/install/docker_registry.html[Docker registry].
```

Comment:
The Architecture topic on the integrated registry (https://docs.openshift.com/enterprise/latest/architecture/infrastructure_components/image_registry.html#integrated-openshift-registry) may be a better target here.